langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaClient.java

```diff
@@ -200,6 +209,23 @@ public OllamaModelCard showInformation(ShowModelInformationRequest showInformati
         }
     }
 
+    private Interceptor headerInterceptor(Map<String, String> customHeaders) {
+        return new Interceptor() {
```

Comment:
please make a copy of `customHeaders` in this method and use it instead of the original map

Reply:
done.